components/Navigation/NavigationContainer/index.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import NavigationContainer from './';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof NavigationContainer>;
+type Meta = MetaObj<typeof NavigationContainer>;
+
+export const Default: Story = {
+  args: {
+    isOpen: false,
```

Comment:
Not sure if this is working, as on the Story if I change isOpen to true or false nothing changes.

Reply:
Okay, I see that this is only for mobile views. I would recommend that we add `https://www.npmjs.com/package/@storybook/addon-viewport` to the Storybook config so that we can test thew view on mobile sizes.

Reply:
i had added. Something else have to be setup. Like viewport ...